src/node.cc

```diff
@@ -2929,6 +2929,12 @@ static void PrintHelp() {
 #endif
          "  --enable-ssl2        enable ssl2\n"
          "  --enable-ssl3        enable ssl3\n"
+         "  --cipher-list=val    specify the default TLS cipher list\n"
+         "  --enable-legacy-cipher-list=val \n"
+         "                       set to v0.10.38 to use the v0.10.38 list,\n"
+         "                       set to v0.10.39 to use the v0.10.39 list.\n"
+         "                       set to v0.12.2 to use the v0.12.2 list.\n"
+         "                       set to v0.12.3 to use the v0.12.3 list.\n"
```

Comment:
I think it would be enough to simply list possible options. users could probably figure out from context that each one corresponds to a given version.